src/main/java/org/fundacionjala/pivotal/core/restapi/RequestManager.java

```diff
@@ -109,4 +109,27 @@ public static Response delete(String endpoint) {
                 .when()
                 .delete(endpoint);
     }
+
+    /**
+     * This method perform a POST or PUT request based in Request Type, endpoint and body map parameters.
+     *
+     * @param requestType The request type.
+     * @param endpoint    is the specified end point.
+     * @param body        is the map body content.
+     * @return a response object.
+     */
+    public static Response requestPostOrPut(RequestType requestType, String endpoint, Map<String, String> body) {
+        return RequestType.POST.equals(requestType) ? post(endpoint, body) : put(endpoint, body);
+    }
+
+    /**
+     * This method perform a GET or DELETE request based in an endpoint parameter.
+     *
+     * @param requestType The request type.
+     * @param endpoint    is the specified end point.
+     * @return is the body content.
+     */
+    public static Response requestGetOrDelete(RequestType requestType, String endpoint) {
```

Comment:
I think those methods should be directly implemented in the step definition
we don't need to wrap it in the RequestManager

Reply:
`String endpoint = DataInterpreter.builtEndPoint(param);`
`response = RequestType.POST.equals(method) ? RequestManager.post(endpoint, map) 
                : RequestManager.put(endpoint, map);`
Like this?

Reply:
I agree, with that change the awkward name method is removed.